cfgov/jinja2/v1/_includes/molecules/breadcrumbs.html

```diff
@@ -17,8 +17,9 @@
     <nav class="breadcrumbs" aria-label="Breadcrumbs">
         {% for crumb in breadcrumbs %}
         <a class="breadcrumbs_link" href="{{ crumb.href if crumb.href else get_protected_url(crumb) }}">
+            {{ svg_icon('left') }}
             {{ crumb.title | e }}
         </a>
         {% endfor %}
     </nav>
-{% endmacro %}
+{% endmacro %}
```

Comment:
Trailing newlines should be retained https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline